include/ibamr/IBFEInstrumentPanel.h

```diff
@@ -100,15 +99,25 @@ class IBFEInstrumentPanel
                             double data_time);
 
     /*!
-     * \brief output exodus files and .dat file for meshes and nodes.
+     * \return get the instrument dump interval.
      */
-    void outputMeterMeshes(IBAMR::IBFEMethod* ib_method_ops, int timestep_num, double data_time);
-
+    int getInstrumentDumpInterval() const;
+    
     /*!
-     * \return return the instrument dump interval.
+     * \return get a pointer to the jjth meter mesh
      */
-    int getInstrumentDumpInterval() const;
-
+    libMesh::SerialMesh* getMeterMesh(const unsigned int jj) const;
```

Comment:
It looks like the meshes are created only once, and so it should be safe to replace this return type by `libMesh::MeshBase&`.

Reply:
By the way, since this is indexed by the meter number, there needs to be a method for finding out the number of meters (e.g., `getNumberOfMeterMeshes()`). It doesn't look like such a function currently exists in the interface.